### ISSUE TITLE
channel diff shows update with no changes

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#8825]
## Observed behavior

The channel update diff from version 95 to 96 showed no updates:

> ![image](https://user-images.githubusercontent.com/2367265/145077042-4d4b098e-86b7-4bb5-8a74-4b0b182ca7da.png)

Presumably, if Studio allowed a new version to be published then _something_ changed. If that's not the case we should open an issue over there.

## Expected behavior

If there was a change that can't be described by one of the existing categories of changes (new, deleted, updated resources) we may need a new row e.g. "channel metadata updated". This doesn't need to be a count, but could be just a statement.

## User-facing consequences

confusing diff on upgrade

## Steps to reproduce

* Inspect the change that occurred in version 96 of `nakav-mafak`
* Replicate and attempt upgrade

## Context

0.15.0-beta3

This will be resolved by:

```[tasklist]
### Tasks
- [ ] Adding a flag for 'metadata updated' - one thing to note is that this will be a significant technical challenge to implement for a very small reward, so this will be very low priority.
- [ ] Properly flagging when supplemental files have changed/been updated
```

**Comment by indirectlylit** [comment]
possibly a duplicate of https://github.com/learningequality/kolibri/issues/6230 but I am not 100% sure

**Comment by radinamatic** [comment]
Agreed with the suggestion to add the new category of  `Channel metadata updated`, to reflect the cases where the resource file was not changed, just title, description, etc (as was the case in this new version of the QA channel). I keep forgetting if changed thumbnail falls into this category...? 🤔 

Either that, or including metadata changes in the `Resources to be updated` category.

**Comment by lauradanforth** [comment]
Flagging that I just experienced this as well....

1. I added captions to existing videos. Studio didn't recognize that changes had been made to the channel so I made a tweak to the title of the resource and that triggered the "Publish" button
2. After the channel was published, I tried to update the channel on Kolibri. It recognized that there was a new version, but showed that 0 resources had changes:
<img width="1061" alt="Screen Shot 2022-02-21 at 4 59 27 PM" src="https://user-images.githubusercontent.com/22602993/155045612-9afe8b23-ed8a-4024-ba53-1c5c268094c8.png">

The import is failing repeatedly on the demo site so I can't verify yet whether the captions made it in or not through the update.

**Comment by nucleogenesis** [comment]
Piling on this issue - I had the same thing happen to me.

I removed some files from a channel, added an exercise, and now the exercise is never recognized by Kolibri, specifically on the BCK instance vodafone-staging -- wondering if this might be related to BCK / Postgres @DXCanas ?

I added another file and it loads fine on the BCK instance, but the exercise doesn't come in. 

Occurs both in UI and when importing from CLI.

I wonder if @lauradanforth 's experience was on a BCK instance too?

**Comment by DXCanas** [comment]
I don’t really know enough about Kolibri’s innards to say whether or not this is a Postgres/BCK thing @nucleogenesis 

do we have evidence that this works with local instances/MySQL? Does redis feed into this at all? Cc @rtibbles
